### PR TITLE
Too many code improvements

### PR DESCRIPTION
**Pull request by dlsf**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java [hunk 1]
```diff
@@ -24,20 +26,20 @@ public CommandManager(String command) {
...
-            try {
-                field.setAccessible(true);
-                com.iridium.iridiumskyblock.commands.Command command = (com.iridium.iridiumskyblock.commands.Command) field
-                    .get(IridiumSkyblock.commands);
-                registerCommand(command);
-            } catch (IllegalAccessException exception) {
-                exception.printStackTrace();
-            }
-        }
+        List<String> manuallyRegisteredCommands = Arrays.asList("shopCommand");
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java:29]: I get what your trying todo here, but I feel  like its making everything a bit more complex.

File: src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java [hunk 2]
```diff
@@ -24,20 +26,20 @@ public CommandManager(String command) {
...
+            .filter(field -> !manuallyRegisteredCommands.contains(field.getName()))
+            .map(field -> {
+                try {
+                    return (com.iridium.iridiumskyblock.commands.Command) field.get(IridiumSkyblock.commands);
+                } catch (IllegalAccessException exception) {
+                    exception.printStackTrace();
+                    return null;
+                }
+            })
+            .filter(Objects::nonNull)
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java:41]: Dont we also need to filter if the command is actually enabled?

File: src/main/java/com/iridium/iridiumskyblock/configs/Commands.java [hunk 3]
```diff
@@ -1,6 +1,56 @@
 package com.iridium.iridiumskyblock.configs;
 
-import com.iridium.iridiumskyblock.commands.*;
+import com.iridium.iridiumskyblock.commands.AboutCommand;
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/configs/Commands.java:3]: Do we need to add each one individual here? We are litrally using everything in the package so a * import should be alright?

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/Commands.java (hunk 3 above)
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/configs/Commands.java:3]: IntelliJ did this, but you are right. I'll change it

File: src/main/java/com/iridium/iridiumskyblock/Island.java [hunk 4]
```diff
@@ -283,87 +264,57 @@ public void initBlocks() {
...
-                                        return original + 1;
-                                    });
-                                }
+                computeValue(plugin, x, z, chunk);
+            }
+        }
+        Bukkit.getScheduler().runTaskLater(IridiumSkyblock.instance, this::calculateIslandValue, 20);
+    }
+
+    private void computeValue(IridiumSkyblock plugin, int finalX, int finalZ, Chunk netherChunk) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/Island.java:285]: Shouldnt this be called chunk insttead since its not necciserily from the nether

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/Island.java:285]: IntelliJ did this, but you are right. I'll change it

File: src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java (hunk 2 above)
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java:41]: We currently don't do that: https://github.com/IridiumLLC/IridiumSkyblock/blob/master/src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java

File: src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java (hunk 1 above)
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java:29]: But open, also Peaches approved my code in the Discord (no one will ever read this lol)

**Pull request merged**